examples/cactus-example-supply-chain-frontend/src/app/bamboo-harvest/bamboo-harvest-detail/bamboo-harvest-detail.page.ts

```diff
@@ -52,9 +52,9 @@ export class BambooHarvestDetailPage implements OnInit {
     });
   }
 
-  public onClickFormSubmit(value: any): void {
+  public onClickFormSubmit(value: unknown): void {
     this.log.debug("form submitted", value);
-    this.bambooHarvest = value;
+    this.bambooHarvest = value as BambooHarvest;
```

Comment:
@adrianbatuto Please add a user defined type guard (if not already present in the codebase) and use that to assert that `value` is the type of `BambooHarvest` at **runtime**. If it's not that type, then throw an error. That way the casting becomes unnecessary.

The casting here is not an optimal solution because it just moves around the problem the linter was talking about to begin with (lack of runtime type assertions)

Reply:
@adrianbatuto The user defined type guard works but now you can remove the casting (which was the whole point)